src/parser/jobs/ast/index.tsx

```diff
@@ -34,7 +34,7 @@ export const ASTROLOGIAN = new Meta({
 	},
```

Comment:
Is there something else still extant to finish wrt supporting AST for 7.0 to 7.05? If not, should we be marking the job supported now?

Reply:
technically an update to arcana logs that sushi has started and I think she wanted to do a more thorough review, but otherwise, I think it's good